docs/en/ingest-management/fleet/add-fleet-server-on-prem.asciidoc

```diff
@@ -98,6 +98,9 @@ You may need to allow access to these ports. Refer to the following table for de
 | {fleet-server} → {es} | 9200
 |===
 
+Note: Connectivity to {kib} on port 5601 is optional and not required at all times. {agent} and {fleet-server} may need to connect to {kib} if deployed in a 
```

Comment:
```suggestion
NOTE: Connectivity to {kib} on port 5601 is optional and not required at all times. {agent} and {fleet-server} may need to connect to {kib} if deployed in a 
```

Here's how it'll render:

<img width="966" alt="Screenshot 2025-01-20 at 9 45 47 AM" src="https://github.com/user-attachments/assets/158c2782-cf98-4758-b65d-721a26efbb4d" />